<table align="center">
  <td>
<a href="https://colab.research.google.com/github/marco-canas/algebra/blob/main/Porcentage1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
        </td>
</table>

# [Video de apoyo a la lectura experimental e interactia de este cuaderno]()

# Aplicaciones de las matemáticas a la economía

En los ejemplos y ejercicios que siguen planteamos ecuaciones que modelan problemas en muchas situaciones distintas de la vida cotidiana.  

## Ejemplo 2 Interés de una inversión
María Camila hereda 450 millones de pesos y los invierte en dos certificados de depósito a término. Uno
de los certificados paga el 6% y el otro paga $ 4.5 \%$ de interés anual **simple**. Si el
interés total de María Camila es 22'491.900 pesos por año, **¿cuánto dinero está invertido en cada tasa?**

## Conceptos asociados al problemas 

* La propiedad distributiva de la multiplicación con respecto a la suma.

* Interés simple e interés compuesto. 

* Propiedad de la balanza o propiedad uniforme. 

* Transponer términos y factores. 


## Establecimiento de la incognita del problema

$x$ la cantidad de dinero de la herencia invertodo al $6\%$ efectivo anual. 

In [5]:
5025*4476
# para pasar de dólares a pesos colombianos multiplicamos por el valor en pesos de cada dolar

22491900

## Modelo en forma verbal:

$$ \text{dinero invertido al $6\%$} + \text{doinero invertido al $4\frac{1}{2}\%$} = \text{total de intereses recibidos} $$


El modelo de esta situción económica es:

$$ x(0.06) + (450-x)(0.045) = 22.491 $$


In [2]:
from sympy import * 

x = var('x') 
modelo = Eq(x*0.06+ (450-x)*0.045, 22.4919)
solve(modelo, x)[0]

149.460000000000

In [3]:
450-solve(modelo, x)[0]

300.540000000000

Lo invertido al 6% efectivo anual es 149'460.000 y lo invertido al 4.5% es 300'540.000 y me parece muy desacertado lo que hizo María Camila pues es mejor invertir más a una tasa más alta de interés. 

In [2]:
300*0.06    # para determinar el 6 porciento de 300 multiplicamos por 0.06

18.0

In [5]:
150*0.045

6.75

In [6]:
300*0.06+150*0.045

24.75

Note que hubiera ganado casi 25 millones de interés si hubiera invertido mas dinero a una tasa más alta. 

# Animación sobre creación de triángulos semejantes

In [4]:
import numpy as np 
import matplotlib.pyplot as plt 



alpha = np.arange(1,2.1, 0.1)
alpha

array([1. , 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 2. ])

## Referentes 

* 